llvm/unittests/CASObjectFormats/BlockDataTest.cpp

```diff
@@ -43,7 +43,7 @@ TEST(BlockDataTest, ZeroFill) {
   };
   for (StringRef ContentHint : {"1", "longer content\n"}) {
     for (uint64_t Size :
-         {1ULL, 2ULL, 7ULL, 1024ULL, uint64_t(ContentHint.size())}) {
+         {1ULL, 2ULL, 7ULL, 1024ULL, (unsigned long long)ContentHint.size()}) {
```

Comment:
Add `#include <cstdint>` instead?

Reply:
> 

The problem here is not `uint64_t` is not defined but compiler refuses to do the list initialization because `uint64_t` and `unsigned long long` from number literal do not match on linux.

Reply:
Oh I see!